### DISCUSSION TITLE
How I can disable the multi-select option

### DISCUSSION THREAD

**Discussion by khalidsharif** [Budibase/budibase#12543] [Help] [answered]
I have an option field in the side panel. it is showing me the options that I need. However, I don't want the user to be able selecting multiple options at the same time ..  I want something like a radio button, only one option is allowed to select.

![image](https://github.com/Budibase/budibase/assets/50369621/35e447ff-f4e0-4046-b8fc-3501d07096fb)

**Comment by melohagan** [comment, marked as answer]
Hey @khalidsharif 

It sounds to me that you want to use the [Options](https://docs.budibase.com/docs/text#options) type and **not** the multi-select type.
